tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -33,6 +33,13 @@ Verify Content In RHODS Explore Section
     Check Number Of Displayed Cards Is Correct  expected_data=${EXP_DATA_DICT}
     Check Cards Details Are Correct   expected_data=${EXP_DATA_DICT}
 
+Verify logged in users are displayed in the dashboard
+    [Documentation]   It verifies that logged in users username is displayed on RHODS Dashboard.
+    [Tags]   Sanity   ODS-354
+    Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
```

Comment:
it is not necessary to repeat line 39-40 because they are included into the `Dashboard Test Setup`

Reply:
I have removed them.